plugins/shared/PluginApi.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace PluginShared;
+
+/// <summary>
+/// This class represents the plugin API specification.
+/// </summary>
+public class PluginApi
+{
+    /// <summary>
+    /// The API specification
+    /// </summary>
+    public string Type { get; set; } = "openapi";
+
+    /// <summary>
+    /// URL used to fetch the specification
+    /// </summary>
+    public string Url { get; set; } = string.Empty;
```

Comment:
This can be done in a later PR but we should store the URL for the manifest, not the Open API spec.

I know we don't use any other fields from the manifest now, but there's work scheduled to leverage the other properties from the manifest (i.e., name for model and description for model) to make plugin discovery / use more effective.

Reply:
This is the Url required by the manifest. We don't store this. this class is used to generate the manifest by code as opposed to a json file..